src/content/api/asset-transfers.mdx

```diff
@@ -22,6 +22,81 @@ their phone number.
 After an asset transfer is completed the recipientAlias, lastSentTo and
 message fields are scrubbed to avoid storing PII.
 
+## Model
+
+### Attributes
+<Properties>
+  <Property name="id" type="string">
+    The unique identifier.
+  </Property>
+
+  <Property name="status" type="string">
+    The status of the Asset Transfer. See [Asset Transfer lifecycle](#asset-transfer-lifecycle) for possible values.
+  </Property>
+
+  <Property name="value" type="bignumber">
+    Amount to send. Required for money transfers. Units depend on the asset type.
+  </Property>
+
+  <Property name="assetId" type="string">
+    Id of a discrete asset to transfer or wallet to draw from.
+  </Property>
+
+  <Property name="assetType" type="string">
+    The [Asset Type](/api/asset-types/) being transferred.
+  </Property>
+
+  <Property name="description" type="string">
+    Shows up in transaction history against the transfer. 200 character limit.
+  </Property>
+
+  <Property name="message" type="string">
+    Shows up in transaction history against the transfer. 100 character limit.
+  </Property>
+
+  <Property name="lastSentTo" type="string">
+    The phone number or email address of the last recipient.
+  </Property>
+
+  <Property name="senderAlias" type="string">
+    Phone number, email or handle of sender.
+  </Property>
+
+  <Property name="senderName" type="string">
+    Human readable name for the sender. 30 character limit.
+  </Property>
+
+  <Property name="recipientAccountId" type="string">
+    The Centrapay Account id of the recipient.
+  </Property>
+
+  <Property name="senderAccountId" type="string">
+    The Centrapay Account id of the sender.
+  </Property>
+
+  <Property name="claimedByAccountId" type="string">
+    The Centrapay Account that claimed the Asset Transfer.
+  </Property>
+
+  <Property name="recipientAlias" type="string">
+    Phone number, email or handle of receiver.
+  </Property>
+
+  <Property name="createdAt" type="timestamp">
+    When the Asset Transfer was created.
+  </Property>
+
+  <Property name="updatedAt" type="timestamp">
+    When the Asset Transfer was updated.
+  </Property>
+
+  <Property name="suppressNotification" type="boolean">
+    Suppress notifications from Centrapay (SMS/Email).
+  </Property>
```

Comment:
@AndyClifford I think we might be missing URL which was documented but perhaps not implemented yet.

Basing this off Seb's demo and this description:
> Some assets can be transferred without supplying a recipient. A `url` field will be
  returned in these cases. The `url` will link to a page to claim the asset.

Reply:
Nice catch needed to pull in latest changes 😅